dist/source/repo-creator/choose-repository.ts

```diff
@@ -224,8 +233,12 @@ export class ChooseRepository {
 
 	private fetchPopular = (): void => {
 		this.repoCreator.getPopular().then((repos: RepositoryWireModel[]) => {
-			let popularTemplates = underscore(repos).map((repo: RepositoryWireModel) => new Repository(repo.owner, repo.name, false, false, true, false));
-			this.mergeTemplates(popularTemplates);
+			//let popularTemplates = underscore(repos).map((repo: RepositoryWireModel) => new Repository(repo.owner, repo.name, false, false, true, false));
+			//this.mergeTemplates(popularTemplates);
+			//let popularTemplates = underscore([{provider:'GitHub',owner:'Foo',name:'Bar'}]).map((repo: RepositoryWireModel) => new Repository(repo.owner, repo.name, false, false, true, false));
+		    //this.mergeTemplates(popularTemplates);
```

Comment:
This line should use tabs for indent, but it currently has a mix of tabs and spaces.